crafts.lua

```diff
@@ -110,25 +110,103 @@ core.register_craft({
 	recipe = { "default:dry_grass_1", "default:dirt" },
 })
 
+-- the density of ethereal mod charcoal is ~1/10 of coal, otherwise it's pure carbon
+-- the volume of charcoal mod charcoal is 1/8 of ethereal mod charcoal, since it uses planks for 2x the output count
+technic.register_compressor_recipe({ input = {"charcoal:charcoal_lump 80"}, output = "default:coal_lump 1" })
```

Comment:
If I remember correctly this had been discussed somewhere and it was decided that we didn't want to be able to convert wood into diamonds.